src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { getCommentsForArticle } from "../utils/api";
+import Comment from "./Comment";
+
+const ArticleComments = ({ article_id }) => {
+  const [articleComments, setArticleComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getCommentsForArticle(article_id)
+      .then((apiComments) => {
+        setArticleComments(apiComments);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setIsError(true);
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <h2>Loading Comments...</h2>;
+  }
+  if (isError) {
+    return <h2>Comments failed to load...</h2>;
+  }
+
+  return (
+    <div>
+      <h2>Comments:</h2>
+      <form>
```

Comment:
Posting comments is two tickets away, please don't add code that is outside the scope of the task you're on.